x11-libs/gtkdatabox/gtkdatabox-1.0.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Gtk+ Widgets for live display of large amounts of fluctuating numerical data"
+HOMEPAGE="https://sourceforge.net/projects/gtkdatabox/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples +glade"
+
+RDEPEND="
+	dev-libs/atk
+	dev-libs/glib:2
+	media-libs/harfbuzz:=
+	x11-libs/cairo
+	x11-libs/gtk+:3
+	x11-libs/gdk-pixbuf:2
+	x11-libs/pango
+	glade? ( dev-util/glade:3.10= )
+"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	default
+
+	# Remove -D.*DISABLE_DEPRECATED cflags
+	find . -iname 'Makefile.am' -exec \
+		sed -e '/-D[A-Z_]*DISABLE_DEPRECATED/d' -i {} + || die
+	# Do Makefile.in after Makefile.am to avoid automake maintainer-mode
+	find . -iname 'Makefile.in' -exec \
+		sed -e '/-D[A-Z_]*DISABLE_DEPRECATED/d' -i {} + || die
```

Comment:
This shouldn't even be needed? But a patch would be nicer for all of this.

(you shouldn't have to touch .in if doing .am)